samples/README.md

```diff
@@ -6,6 +6,7 @@ These samples demonstrate how to use the Oboe library:
 sine wave when you tap the screen
 1. [RhythmGame](RhythmGame): A simple rhythm game where you copy the clap patterns you hear by tapping on the screen
 1. [MegaDrone](MegaDrone): A one hundred oscillator synthesizer, demonstrates low latency and CPU performance
+1. [LiveEffect](LiveEffect): loops back audio from input stream to output stream to demonstrate duplex capability
```

Comment:
Thing is, it's not really a "loop back" as this implies a signal path which circles back on itself. Instead what we have is: 

recording at microphone -> app processing -> playback at speaker

Needs rewording

Reply:
yeah, that is right no "back", just a loop: drop the "back". for loopback I thought Phils cts coins the same term, am I correct?